recipes/mesa/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+autoreconf -fi
+
+./configure \
+    CXXFLAGS="-O2 -g -DDEFAULT_SOFTWARE_DEPTH_BITS=31" \
+    CFLAGS="-O2 -g -DDEFAULT_SOFTWARE_DEPTH_BITS=31" \
+    --disable-xvmc \
+    --disable-glx \
+    --disable-dri \
+    --with-dri-drivers="" \
+    --with-gallium-drivers="swrast" \
+    --enable-texture-float \
+    --disable-shared-glapi \
+    --disable-egl \
+    --disable-gles1 \
+    --disable-gles2 \
+    --with-egl-platforms="" \
+    --enable-gallium-osmesa \
+    --enable-gallium-llvm=yes \
+    --with-llvm-shared-libs \
+    --prefix=$PREFIX
+
+make -j2
+make -j4 install
```

Comment:
Don't use the `2` and `4` here because we don't really know how many core we can get on the CIs. If you are feeling adventurous though you can use `-j${CPU_COUNT}` and use the current "known" number of cores.